index.html

```diff
@@ -44,5 +45,152 @@ <h2>LET’S CONNECT</h2>
         </ul>
       </div>
     </section>
+
+    <!--Works section-->
+    <section id="works">
+      <ul class="works-container">
+        <li class="work-card">
+          <img src="/media/snapshoot-portfolio.png" alt="Work preview for project Tonic" class="work-card-img">
+          <h1 class="work-card-tittle">Tonic</h1>
+          <div class="work-card-info">
+            <h2>Canopy</h2>
+            <span class="circle"></span>
+            <h3>Back End Dev</h3>
+            <span class="circle"></span>
+            <h3>2015</h3>
+          </div>
+          <p class="work-card-description">
+            A daily selection of privately 
+            personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="work-card-tags">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javaScript</li>
+          </ul>
+          <button type="button" class="work-card-seeProject">See Project</button>
+        </li>
+
+        <li class="work-card">
+          <img src="/media/snapshoot-portfolio-1.png" alt="Work preview for project Multi-Post Stories" class="work-card-img">
+          <h1 class="work-card-tittle">Multi-Post Stories</h1>
+          <div class="work-card-info">
+            <h2>Canopy</h2>
+            <span class="circle"></span>
+            <h3>Back End Dev</h3>
+            <span class="circle"></span>
+            <h3>2015</h3>
+          </div>
+          <p class="work-card-description">
+            A daily selection of privately 
+            personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="work-card-tags">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javaScript</li>
+          </ul>
+          <button type="button" class="work-card-seeProject">See Project</button>
```

Comment:
- [ ] As suggested by the last reviewer, setting the width property seems not to be solving the issue because the button is still taking full width. Kindly find a way to adjust the width of the button such that the width is limited to fit the content of the button. Doing this will ensure that your page matches what we have in the design :blush:

Reply:
I don't know if this is proper, let me know... This is a screenshot of the current code with the width: fit-content; and it is working properly for me.

![image](http://imgur.com/tcrbteQl.png)

Reply:
Hi @D4nielJ,

I've rechecked it using the local live server and it's working well. However, I tried using `https://raw.githack.com/` to check the live version and it's not working. I also realized that this is applicable to the images too. To avoid confusion, I suggest you should host your project, either with GitHub pages or the link above, and ensure that everything is working well before you submit another review request. 

Do not forget to add the live demo link to your Readme :relaxed: